tests/reload/20-stop-point/reference.log

```diff
@@ -0,0 +1,7 @@
+2016-07-08T14:53:12+01 INFO - Initial point: 1
+2016-07-08T14:53:12+01 INFO - Final point: 5
+2016-07-08T14:53:12+01 INFO - [set-stop-point.1] -triggered off []
+2016-07-08T14:53:19+01 INFO - [reload.1] -triggered off ['set-stop-point.1']
```

Comment:
This highlights the usefulness of a revamped suite log as outlined in #1032 - with a line (or a note in the 'stopping' notification near the end) conveying the setting of a stop point via a ``cylc stop <suite name> <stop point>`` command it would be clear the suite hasn't shutdown erroneously before the FCP.